src/twisted/test/test_sslverify.py

```diff
@@ -206,7 +206,9 @@ def certificatesForAuthorityAndServer(serviceIdentity=u'example.com'):
     try:
         ipAddress = ipaddress.ip_address(serviceIdentity)
     except ValueError:
-        subjectAlternativeNames = [x509.DNSName(serviceIdentity)]
+        subjectAlternativeNames = [
+            x509.DNSName(serviceIdentity.encode("idna").decode("ascii"))
```

Comment:
This is resolving a different deprecation. It looks right but I don’t know what your PR policies are on that.

Reply:
Given that this would also be a .misc it’s nbd; if the change were something meriting is own news fragment it would need to be a different PR.